src/controllers/signup.js

```diff
@@ -0,0 +1,4 @@
+exports.get = (req, res) => {
+  res.render('signup' , { layout:'login_signup', title : 'signup ' , 
```

Comment:
@Fatmasiam This isn't an urgent change, but it would look a little prettier if the second argument object was formatted more like this:
```
res.render('signup' , {
  layout:'login_signup',
  title : 'signup ' , 
  js:['collectData', 'signup'], 
  css: ['signup']
})
```
It might just be my personal preference 🤷‍♂️

Reply:
@tomduggan85  
thanx, that's better and more clear to read